### DISCUSSION TITLE
Access violation at address 534F6B9B in module 'libASAP2.dll'. Read of address 00000000

### DISCUSSION THREAD

**Discussion by bluesilhouettes** [TOSUN-Shanghai/TSMaster#110] [Q&A]
Dear TSmaster team,

I got this error everytime trying to import and a2l file.

Do you have any hint for me to overcome it ?

[https://drive.google.com/file/d/1ukNvl3yHWXyrBG16yDL_ZCiWc7xXWkOM/view?usp=sharing](url)

**Comment by freshhope** [comment]
could you send me the a2l file to checkout this issue? Thank you.
